packages/client/src/execution/vmexecution.ts

```diff
@@ -429,7 +611,25 @@ export class VMExecution extends Execution {
                       td,
                       timestamp,
                     })
+                    const isPostPrague = this.config.execCommon.gteHardfork(Hardfork.Prague)
+                    if (wasPrePrague && isPostPrague) {
+                      await this.transitionToVerkle(parentState!)
+                      clearCache = false
+                    }
```

Comment:
Ok, but seeing all this Prague-centered logic here: maybe we keep Prague, right?

Reply:
my opinion is that we keep client with HF centric logic while downstream libs should should EIP enabled centic logic

Reply:
makes sense